### PR TITLE
new commit

### PR DESCRIPTION
**Pull request by DmtrVoloshyn**

### CODE REVIEW

File: Homework/Homework4/Program.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    internal class Program
+    {
+        private static void Main(string[] args)
+        {
+            Console.Write("Enter the number of array elements: ");
+            var input = Console.ReadLine();
+            int arrayLenght = int.TryParse(input, out int result) ? result : 0;
+
+            int[] array = ArrayGenerator.GenerateValues(arrayLenght, 1, 26);
+            (int[] evenArray, int[] oddArray) = ArrayProcessor.DivideIntoEvenAndOdd(array);
```
Comment (PeterChernykh) [inline Homework/Homework4/Program.cs]: Better to create model for this purpose or create method in which we can send input parameter of tje type we need either odd or even

File: Homework/Homework4/Program.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                Console.WriteLine($"Letters array from the even numbers has more capital letters: {countUpperLettersInArrayFromEvenNumbers}");
+            }
+            else if (countUpperLettersInArrayFromEvenNumbers < countUpperLettersInArrayFromOddNumbers)
+            {
+                Console.WriteLine($"Letters array from the odd numbers has more capital letters: {countUpperLettersInArrayFromOddNumbers}");
+            }
+            else
+            {
+                Console.WriteLine($"Both arrays include an equal number of capital letters. First - {countUpperLettersInArrayFromEvenNumbers} and second {countUpperLettersInArrayFromOddNumbers}");
+            }
```
Comment (PeterChernykh) [inline Homework/Homework4/Program.cs:46]: Can be moved to a separate method

File: Homework/Homework4/StringCheckLibrary/Class1.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
+﻿namespace StringCheckLibrary
+{
+    public class StringChecker
+    {
+        private static readonly string[] ValidStrings = { "a", "e", "i", "d", "h", "j" };
```
Comment (PeterChernykh) [inline Homework/Homework4/StringCheckLibrary/Class1.cs]: We can use char here

Comment (DmtrVoloshyn) [reply Homework/Homework4/StringCheckLibrary/Class1.cs]: fixed in next commit

File: Homework/Homework4/Program.cs (hunk 2 above)
Comment (DmtrVoloshyn) [reply Homework/Homework4/Program.cs:46]: fixed in next commit

File: Homework/Homework4/Program.cs (hunk 1 above)
Comment (DmtrVoloshyn) [reply Homework/Homework4/Program.cs]: fixed in next commit

**Pull request merged**